spec/requests/checkout/failed_checkout_spec.rb

```diff
@@ -23,6 +23,10 @@
   end
 
   before do
+    order_cycle_distributed_variants = double(:order_cycle_distributed_variants)
+    allow(OrderCycleDistributedVariants).to receive(:new).and_return(order_cycle_distributed_variants)
+    allow(order_cycle_distributed_variants).to receive(:distributes_order_variants?).and_return(true)
+
```

Comment:
I spent over an hour trying to make these tests pass by adding the test variants to the order cycle but got really stuck with it (and the 14, yes 14, before_filters on the checkout controller do not help).
I decided to go for mocking... I dont think it's ideal to mock a specific class when testing the controller but it works.